### PR TITLE
carefully wrap DOM element to fix #3076

### LINKED ISSUE #3076: FlowCrypt does not recognize signatures from recent Thunderbird versions
Thunderbird implemented their own OpenPGP now, and will stop relying on Enigmail.

There is something about their signatures that causes FlowCrypt to not realize the messages are signed.

Samples available on flowcrypt.compatibility@gmail.com account.

image

Use this to bring the sample into our mock tests:
```
  /**
   * This is the proper way to add messages to mock api for testing:
   *   1) log into flowcrypt.compatibility@gmail.com
   *   2) go to Settings -> Inbox and find your message
   *   3) click "download api export"
   *   4) save the json file to exported-messages folder
   */
  private exportedMsgsPath = './test/source/mock/google/exported-messages/';
```

### PR DESCRIPTION
**Pull request by rrrooommmaaa**
Fixes #3076

### CODE REVIEW

**Review by tomholub — COMMENTED** [review]
Test looks great. The fix is hard to follow but this is normal when fiddling with the Gmail DOM. Looks good - as long as it works.

**Comment by rrrooommmaaa** [comment]
@tomholub The problem was that `msgBody.html()` returned `div` element without `id` attribute, while with DOM manipulations it was possible to preserve it.

**Review by tomholub — DISMISSED** [review]

**Comment by rrrooommmaaa** [comment]
@tomholub Let me add the second (plain) message to the test

**Review by tomholub — APPROVED** [review]
